src/vs/workbench/parts/watermark/electron-browser/watermark.ts

```diff
@@ -18,7 +18,6 @@ import { IWorkspaceContextService } from 'vs/platform/workspace/common/workspace
 import { IWorkbenchContribution, IWorkbenchContributionsRegistry, Extensions as WorkbenchExtensions } from 'vs/workbench/common/contributions';
```

Comment:
It had a reference to the Action. As I removed the action, this had to be removed as well